.github/workflows/build.yml

```diff
@@ -1,14 +1,12 @@
 name: Build
 on: 
   workflow_dispatch:
-  push:
+  pull_request:
+    types: [closed]
```

Comment:
Is closed == merged?

Reply:
Great question! The associated issue #209  links to this [blog post](https://github.com/orgs/community/discussions/26724). In short no, closed can mean merged or closed w/out merge. I wrote up my thoughts in the issue